x-pack/plugins/ml/public/embeddables/single_metric_viewer/use_single_metric_viewer_input_resolver.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEffect, useState } from 'react';
+import { combineLatest, Observable } from 'rxjs';
+import { startWith } from 'rxjs/operators';
+import { TimefilterContract } from '@kbn/data-plugin/public';
+import { SingleMetricViewerEmbeddableInput } from '..';
+import type { TimeRangeBounds } from '../../application/util/time_buckets';
+
+export function useSingleMetricViwerInputResolver(
```

Comment:
typo `Viwer`

Reply:
Updated in f682daaa0b13b541a66cc5725072a61828502541